packaging/installer/kickstart.sh

```diff
@@ -351,7 +352,7 @@ done
 
 if [ "${INTERACTIVE}" = "0" ]; then
 	PACKAGES_INSTALLER_OPTIONS="--dont-wait --non-interactive ${PACKAGES_INSTALLER_OPTIONS}"
-	NETDATA_INSTALLER_OPTIONS="--dont-wait"
+	NETDATA_INSTALLER_OPTIONS="$NETDATA_INSTALLER_OPTIONS --dont-wait"
```

Comment:
The same here: could you make this so that it doesn't add an extra space when $NETDATA_INSTALLER_OPTIONS is empty?

Reply:
See my other comment